packages/headless/src/features/facets/range-facets/generic/range-facet-controller-actions.ts

```diff
@@ -1,13 +1,11 @@
 import {createAsyncThunk} from '@reduxjs/toolkit';
```

Comment:
I think it's possible to push this refactor further, removing this file, `date-facet-controller-actions`, `numeric-facet-controller-actions`, and all three corresponding test files.

The only dispatch left here is `dispatch(updateFacetOptions({freezeFacetOrder: true}));` which we can duplicate. I don't think it's worth all the indirection,